composer.json

```
@@ -5,11 +5,11 @@
       "name": "Adyen"
     }
   ],
-  "description": "Processes notification for Adyen Payment Plugins",
+  "description": "Webhook module for Adyen Payment Plugins",
```

Not necessarily useful only for plugins, could be used in any kind of PHP integrations.
```suggestion
  "description": "Webhook module for Adyen Payment integrations",
```